.golangci.yml

```diff
@@ -4,23 +4,98 @@ run:
 #   timeout: 5m
 
 linters:
+  # Disable all linters.
+  # Default: false
```

Comment:
nit: I would remove any commented out sections unless we intend to re-enable them in the future.

```suggestion
```